### PR TITLE
Finalize changelog

### PR DESCRIPTION
**Pull request by jaydenmilne**

### CODE REVIEW

File: src/holodeck/sensors.py [hunk 1]
```diff
@@ -593,7 +593,7 @@ class AbuseSensor(HolodeckSensor):
     **Configuration**
 
     - ``AccelerationLimit``: Maximum acceleration the agent can endure from before
```
Comment (vinhowe) [inline src/holodeck/sensors.py]: "Maximum acceleration the agent can endure from" sounds strange&mdash;maybe I'm not reading it right. Maybe just "Maximum acceleration the agent can endure"?

File: docs/changelog/changelog.rst [hunk 2]
```diff
@@ -67,24 +74,44 @@ Changes
 - Removed the ability to toggle sensors during runtime with the removal of
   :class:`~holodeck.environments.command.SetSensorEnabledCommand`,
   :meth:`~holodeck.environments.HolodeckEnvironment.set_sensor_enabled`, and
-  :meth:`~holodeck.sensors.HolodeckSensor.set_sensor_enable`.
```
[the remark this answers is not in the document]
Comment (jaydenmilne) [reply docs/changelog/changelog.rst]: lol OK, funny

[the remark this answers is not in the document]
Comment (jaydenmilne) [reply docs/changelog/changelog.rst]: Changed

File: src/holodeck/sensors.py (hunk 1 above)
Comment (jaydenmilne) [reply src/holodeck/sensors.py]: Deleted word

**Pull request merged**